test/visualData.ts

```diff
@@ -74,12 +72,14 @@ export class VisualData extends TestDataViewBuilder {
                     },
                     values: this.valuesBottomValues
                 }
-            ], columnNames).build();
+            ], <string[]>columnNames).build();
     }
 }
 
 export function getRandomHexColor(): string {
-    return getHexColorFromNumber(getRandomInteger(0, 16777215 + 1));
+    const value = Math.floor(Math.random() * (16777215 + 1))
+    const hex = value.toString(16).toUpperCase();
```

Comment:
no need in uppercase, but it's up to you. 
Also, I would suggest add a comment about the multiplying by 16777215 (which is 0xFFFFFF in hexadecimal) ensures that the random number covers the full range of possible hex colors, but this is also not critical.

Reply:
Added the comment.